ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteExporter.java

```diff
@@ -722,8 +722,10 @@ private static UctePhaseRegulation convertRatioTapChanger(TwoWindingsTransformer
                 twoWindingsTransformer.getRatioTapChanger().getHighTapPosition(),
                 twoWindingsTransformer.getRatioTapChanger().getTapPosition(),
                 Double.NaN);
-        if (!Double.isNaN(twoWindingsTransformer.getRatioTapChanger().getTargetV())) {
-            uctePhaseRegulation.setU(twoWindingsTransformer.getRatioTapChanger().getTargetV());
+        // TODO : add reactive power case ?
```

Comment:
If UCTE format supports it, yes. If not, no.

Reply:
UCTE format does not support reactive power control mode, remove the TODO